cypress/integration/ATATDevSnow/PerformanceRequirements/ComputeGatheringDetails.spec.js

```diff
@@ -0,0 +1,334 @@
+import { 
+  bootstrapMockApis,   
+  randomAlphaNumeric,
+  randomNumber,
+  getObjectFromArrayByKey  
+} from "../../../helpers";
+import common from "../../../selectors/common.sel";
+import contractDetails from "../../../selectors/contractDetails.sel";
+import performanceReq from "../../../selectors/performanceReqs.sel";
+
+describe("Test suite: Let’s start by gathering your Compute requirements", () => {
+
+  let serviceOfferingGroups;
+  let compute;
+  let categoryObj;
+  
+  beforeEach(() => {
+    bootstrapMockApis();
+
+    cy.fixture("serviceOfferingGroups").then((data) => {
+      serviceOfferingGroups = data;
+      categoryObj = getObjectFromArrayByKey(
+        serviceOfferingGroups, "value", "COMPUTE"
+      );
+    });
+    cy.fixture("compute").then((data) => {
+      compute = data;
+      
+    });
+    
+    cy.launchATAT();
+    cy.clickSideStepper(common.stepContractDetailsLink, " Contract Details ");
+    cy.verifyPageHeader(" Let’s gather some details about the duration of your task order ");
+    cy.dropDownClick(contractDetails.baseDropdownIcon);
+    cy.findElement(contractDetails.baseDropdownMonth).click({ force: true });
+    //Enter the Value for Base
```

Comment:
Please remove this comment, or add to it if the `baseInputTxtBox` object is not specific enough

Reply:
Removed the comment